pandas/core/ops/__init__.py

```diff
@@ -329,19 +329,27 @@ def fill_binop(left, right, fill_value):
 
     Notes
     -----
-    Makes copies if fill_value is not None
+    Makes copies if fill_value is not None and NAs are present.
     """
-    # TODO: can we make a no-copy implementation?
     if fill_value is not None:
         left_mask = isna(left)
         right_mask = isna(right)
-        left = left.copy()
+
         right = right.copy()
```

Comment:
Should the right copy not be removed as well?

Reply:
good catch, will update